wallet/createtx.go

```diff
@@ -46,14 +46,15 @@ const (
 	// sanityVerifyFlags are the flags used to enable and disable features of
 	// the txscript engine used for sanity checking of transactions signed by
 	// the wallet.
-	sanityVerifyFlags = mempool.BaseStandardVerifyFlags
+	sanityVerifyFlags = txscript.ScriptDiscourageUpgradableNops |
```

Comment:
the txscript type used here is not aliased between v1 and v2 modules and i was only importing mempool for this single constant.  this could be reverted after mempool/v2 is introduced or we could define the txscript constant to be type compatible by making it an alias.